testool/src/statetest/executor.rs

```diff
@@ -189,6 +191,8 @@ fn into_traceconfig(st: StateTest) -> (String, TraceConfig, StateTestResult) {
                 v: sig.v,
                 r: sig.r,
                 s: sig.s,
+                rlp_bytes: rlp_unsigned,
+                rlp_unsigned_bytes: rlp_signed,
```

Comment:
This should be:
```suggestion
                rlp_bytes: rlp_signed,
                rlp_unsigned_bytes: rlp_unsigned,
```
?

Reply:
fixed in https://github.com/scroll-tech/zkevm-circuits/pull/488/commits/f3f656e20de67c5028dbb1ce1ea0ced30b35c6b2